### DISCUSSION TITLE
Safari can not open the page because the address is invalid

### DISCUSSION THREAD

**Discussion by levgenprog** [ionic-team/capacitor#7610] [Q&A]
I've build the application for ios following the steps in the documentation, and also configured deep (universal links) as also guided in the documentation here: [https://capacitorjs.com/docs/v5/guides/deep-links](https://capacitorjs.com/docs/v5/guides/deep-links). The application opens, but when I try to redirect for the authorization, I get this error on the screen: 
![IMG_364915D4A3A5-1](https://github.com/user-attachments/assets/63140e9f-bb43-4362-bcc1-5720d6274785)

Can anyone tell me what is wrong or how can it be resolved?

**Comment by cellulosa** [comment]
did you ever found a solution?

**Comment by cellulosa** [comment]
For the moment I managed to do it with `InAppBrowser`:

```ts
import { InAppBrowser, DefaultWebViewOptions } from "@capacitor/inappbrowser";

// Build web login URL
const webLoginUrl = new URL('/login', 'http://localhost/etc');
webLoginUrl.searchParams.set('redirectUri, 'my-app://etc');

await InAppBrowser.openInWebView({
    url: webLoginUrl.toString(),
    options: DefaultWebViewOptions,
});
```
